### DISCUSSION TITLE
Ohm v17.1.0

### DISCUSSION THREAD

**Discussion by pdubroy** [ohmjs/ohm#449] [Releases]
[![NPM](https://img.shields.io/npm/v/ohm-js.svg)](https://www.npmjs.com/package/ohm-js)
**Browser bundles:** [ohm.js](https://unpkg.com/ohm-js@17.1.0/dist/ohm.js) • [ohm.min.js](https://unpkg.com/ohm-js@17.1.0/dist/ohm.min.js)

## New features:

- Added `extractExamples` to the `extras` module, which can extract example inputs — written with a special comment syntax — from grammar definitions.

<hr /><em>This discussion was created from the release <a href='https://github.com/ohmjs/ohm/releases/tag/v17.1.0'>Ohm v17.1.0</a>.</em>

**Comment by ynx0** [comment]
Yo this is still alive!! 🥳
